GameData/RealismOverhaul/Engine_Configs/BE-4_Config.cfg

```diff
@@ -111,7 +106,7 @@
 //	TestFlight compatibility.
 //	==================================================
 
-@PART[*]:HAS[@MODULE[ModuleEngineConfigs]:HAS[@CONFIG[BE-4]],!MODULE[TestFlightInterop]]:BEFORE[zTestFlight]
+@PART[*]:HAS[@MODULE[ModuleEngineConfigs]:HAS[@CONFIG[BE4]],!MODULE[TestFlightInterop]]:BEFORE[zTestFlight]
```

Comment:
Appears I meant here.  Was briefly confused.  But the `engineType` is BE4, and the Config is `BE-4`.  So this line should be reverted back to BE-4.

Reply:
https://github.com/KSP-RO/RealismOverhaul/blob/master/GameData/RealismOverhaul/RO_SuggestedMods/VSR/VNP/RO_VNP_Engines.cfg#L340
https://github.com/KSP-RO/RealismOverhaul/blob/master/GameData/RealismOverhaul/RO_SuggestedMods/KK%20Launchers/RO_KK_Vulcan_Engines_ULA.cfg#L112

Reply:
Those set `engineType`.  You are writing TestFlight values for `CONFIG`.  One has a -, the other doesn't.

Reply:
oh ok i get it